File: app/src/main/java/com/wiktor/weater1/newWeaher/cityList/mvp/CityListView.java
package com.wiktor.weater1.newWeaher.cityList.mvp;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;
import com.wiktor.weater1.newWeaher.cityList.model.CityModel;

import java.util.List;

public interface CityListView extends MvpView {

    @StateStrategyType(AddToEndStrategy.class)
    void showCityList(List <CityModel> list);
}

File: app/src/main/java/com/wiktor/weater1/newWeaher/cityList/mvp/CityListView.kt
package com.wiktor.weater1.newWeaher.cityList.mvp

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType
import com.wiktor.weater1.newWeaher.cityList.model.CityModel


interface CityListView : MvpView {

    @StateStrategyType(AddToEndSingleStrategy::class)
    fun showCityList(list: List<CityModel>)
}

